projects/fusion-e2e/entities/tests/chart/chartLabelIcon.pw.ts

```diff
@@ -0,0 +1,50 @@
+import {expect, Locator, test} from '@playwright/test';
+import {ChartPage} from '../../pages/chart-page';
+import {chartLabelWithIconStoryId} from './consts';
+import {CART_LABELS_ICONS_MOCK} from '@ironsource/fusion-ui/components/chart-labels/v4/chart-labels-v4.stories.mock';
+import {toRgba} from '../utils';
+
+const ALPHA_VALUE = 100;
+
```

Comment:
General note - I know you work with icons and I didn’t see you covered any tests that validate them. playwright exposes a pixel validation method you should read about it and implement it - if you need a reference from our project you can check  demand-platform-ui 

https://playwright.dev/docs/test-snapshots